docs/README_DEPLOY.md

```
@@ -131,34 +139,38 @@ graph LR
+| **パッケージ作成(複数版)** | 約 20分 (Release/Dev合計) | **0分**              | アイコン色変え・名称変更のミス排除 |
+| **サイト・リリース公開**   | 約 10分 (FTP/Upload)      | **0分**              | 常に最新版が公開される安心感       |
+| **合計**                   | **約 60分 / 回**          | **0分**              | 本質的な開発時間に集中できる       |
 
 ---
 
 ## Vercel への初回設定方法
-*※すでに設定済みの場合は不要です。*
+
+_※すでに設定済みの場合は不要です。_
 
 #### 1. Vercel での準備
```

markdownlint-cli2 (0.23.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)